iis-common/src/main/java/eu/dnetlib/iis/common/report/PigCounterValueResolver.java

```diff
@@ -0,0 +1,65 @@
+package eu.dnetlib.iis.common.report;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.commons.lang3.StringUtils;
+
+import eu.dnetlib.iis.common.counter.PigCounters;
+
+/**
+ * Resolver of values with pig counters placeholder.
+ * 
+ * @author madryk
+ */
+public class PigCounterValueResolver {
+
+    private static final Pattern PLACEHOLDER_PATTERN = Pattern.compile("\\{(.*)\\}");
+    
+    private static final char PLACEHOLDER_VALUES_SEPARATOR = '.';
+    
+    
+    //------------------------ LOGIC --------------------------
+    
+    /**
+     * Returns resolved value using provided {@link PigCounters}.<br/>
+     * Passed value can be a simple string or string with placeholder.
+     * Simple string will be returned unchanged.<br/>
+     * If value contains placeholder it will be resolved to corresponding
+     * value from {@link PigCounters}. Placeholder must be in form:
+     * <code>{JOB_ALIAS.JOB_COUNTER}</code>.
+     */
+    public String resolveValue(String value, PigCounters pigCounters) {
```

Comment:
Wouldn't it be better for the method to be passed jobAlias and counterName? Why should one pass some strange placeholder string when one could pass just the two necessary values?